numpy/core/function_base.py

```diff
@@ -72,7 +72,8 @@ def linspace(start, stop, num=50, endpoint=True, retstep=False, dtype=None):
     samples : ndarray
         There are `num` equally spaced samples in the closed interval
         ``[start, stop]`` or the half-open interval ``[start, stop)``
-        (depending on whether `endpoint` is True or False).
+        (depending on whether `endpoint` is True or False).  If start or
+        stop are array-like, then the samples will be on the first axis.
```

Comment:
Perhaps "will be along a new axis inserted at the beginning" would be more accurate

Reply:
Adopted.